shell/browser/ui/webui/accessibility_ui.cc

```diff
@@ -391,25 +391,25 @@ void ElectronAccessibilityUIMessageHandler::RequestNativeUITree(
 void ElectronAccessibilityUIMessageHandler::RegisterMessages() {
   DCHECK_CURRENTLY_ON(content::BrowserThread::UI);
 
-  web_ui()->RegisterMessageCallback(
+  web_ui()->RegisterDeprecatedMessageCallback(
```

Comment:
Can we update to use the non-deprecated version ahead of time, so we're prepared when this gets deleted?

Reply:
@nornagon the handlers (ef https://source.chromium.org/chromium/chromium/src/+/main:chrome/browser/accessibility/accessibility_ui.cc;l=444?q=AccessibilityUIMessageHandler::ToggleAccessibility&ss=chromium) invoked here have not been updated, so I don't think we can use the non-deprecated version ahead of time.